### ISSUE TITLE
[bug] Camera and microphone do not work in iframe

### ISSUE THREAD

**Issue by Sunvas** [nwjs/nw.js#8235]
### Current/Missing Behavior
Neither camera nor microphone work in iframe.

### Expected/Proposed Behavior
They should work

### Additional Info
- Operating System: any
- NW.js Version: 0.93 (the latest)
- Code snippet: see below

### How to reproduce

**package.json**
```
{
	"main": "/main.html",
	"name": "demo"
}
```

**main.html**
```
<!doctype html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>Demo</title>
	<script>
chrome.contentSettings.camera.set({
	primaryPattern:"https://webcamtests.com/*",
	scope: "regular",
	setting: "allow"
});
	</script>
</head>
<body style="width:100vw;height:100vh">
	<h2>Camera doesn't work in webviews and iframes:</h2>
	<iframe style="width:100%;height:100%" src="https://webcamtests.com" allow="camera https://webcamtests.com; microphone https://webcamtests.com"></iframe>
</body>
</html>
```

Demo opens [Webcam Test](https://webcamtests.com) which perfectly shows that neither camera nor microphone can be even detected.

**Comment by ayushmanchhabra** [comment]
https://blog.addpipe.com/camera-and-microphone-access-in-cross-oirigin-iframes-with-feature-policy/
A quick internet search brought me to this. Please go through this and see if it helps

**Comment by Sunvas** [comment]
@ayushmanchhabra  No, that doesn't help. Everyhing is correct with my iframe definition: it works in a regular [environment](https://jsfiddle.net/Sunvas/wya8of3h/).

**Comment by Sunvas** [comment]
Clipboard also doesn't work in iframe. Have mentioned this in #8170

**Comment by Sunvas** [comment]
[Solution](https://github.com/nwjs/nw.js/issues/7855#issuecomment-2801734227).
